### PR TITLE
Suggested edits for the tutorial and help.

### PR DESCRIPTION
**Pull request by mshulman**
Instead of setting the max height to 0.075 of the canvas, set it to 0.13 of the canvas. 

This looks a little bigger, and still resizes appropriately.

### CODE REVIEW

**Comment by mshulman** [comment]
Suggestions to the help text  for Configuration Management

File: src/app/widgets/widget-numeric/widget-numeric.component.ts [hunk 1]
```diff
@@ -320,7 +320,7 @@ export class WidgetNumericComponent extends BaseWidgetComponent implements OnIni
 
     let valueText: string = '';
     const maxTextWidth = Math.floor(this.canvasMM.nativeElement.width * 0.45);
-    const maxTextHeight = Math.floor(this.canvasMM.nativeElement.height * 0.075);
+    const maxTextHeight = Math.floor(this.canvasMM.nativeElement.height * 0.13);
```
Comment (godind) [inline src/app/widgets/widget-numeric/widget-numeric.component.ts:323]: It starts big, but goes small again on first data refresh. This is probably not the right place, or the max size looping bellow is not taking into account.

**Review by godind — CHANGES_REQUESTED** [review]

File: src/app/widgets/widget-numeric/widget-numeric.component.ts (hunk 1 above)
Comment (godind) [reply src/app/widgets/widget-numeric/widget-numeric.component.ts:323]: I could have approved the other changes. Try to create PRs that are targeted to one area or feature set. Next time ;)

**Comment by mshulman** [comment]
It looked good on my setup when I tested.

I really need to learn how Angular Canvases work.

-ms

**Comment by godind** [comment]
No worries. That's what PR reviews are for. But they do take time!

**Comment by mshulman** [comment]
Reverted the maxHeight change.

**Pull request merged**